html/modules/custom/docstore/src/SearchableResourceTrait.php

```diff
@@ -233,6 +233,23 @@ public function searchResources(Request $request, $entity_type_id, ?ConfigEntity
         $data['revisions'] = $this->getResourceEntityRevisionList($entity_type_id, $id);
       }
     }
+    else {
+      // Multiple results.
+      $output = [
+        '_count' => $results->getResultCount(),
+      ];
+
+      // Add extra metadata.
+      if ($extra = $results->getExtraData('search_api_solr_response')) {
+        $output['_start'] = $extra['response']['start'];
+        $output['_numFoundExact'] = $extra['response']['numFoundExact'];
```

Comment:
`_numFoundExact` sounds a bit cryptic. What about `_total` or `_totalCount`?

Also not sure if `$extra['response']['numFoundExact']` and `$extra['response']['start']` are always set, for example if there is no matching results.

Reply:
> `_numFoundExact` sounds a bit cryptic

True, it's a boolean indicating that the count is correct or an estimate, can be removed.


> start

Not sure if this is always set and if it will be always correct